west.yml

```diff
@@ -13,3 +13,9 @@ manifest:
       repo-path: sdk-nrf
       revision: v2.7.0
       import: true
+
+    # TODO: Once west supports sparse checkout, we should use it here.
+    - name: nrfxlib
+      remote: ncs
+      repo-path: sdk-nrfxlib
+      revision: pull/1414/head
```

Comment:
as this release is based on NCS 2.7.0, we can do a tag on the PR branch, but in the future if we move to next NCS release 2.8.0, then this would be changed anyway.